plugins/InspectionGadgets/src/com/siyeh/ig/redundancy/RedundantStringOperationInspection.java

```diff
@@ -93,7 +97,9 @@ public PsiElementVisitor buildVisitor(@NotNull ProblemsHolder holder, boolean is
       .register(METHOD_WITH_REDUNDANT_ZERO_AS_SECOND_PARAMETER, this::getRedundantZeroAsSecondParameterProblem)
       .register(STRING_LAST_INDEX_OF, this::getLastIndexOfProblem)
       .register(STRING_IS_EMPTY, this::getRedundantCaseChangeProblem)
-      .register(STRING_EQUALS, this::getRedundantSubstringEqualsProblem);
+      .register(STRING_EQUALS, this::getRedundantSubstringEqualsProblem)
+      .register(STRING_EQUALS, this::getRedundantCaseEqualsProblem)
```

Comment:
I believe, you may use `.register(anyOf(STRING_EQUALS, STRING_EQUALS_IGNORE_CASE), this::getRedundantCaseEqualsProblem)`

Reply:
I changed to .register(anyOf(STRING_EQUALS, STRING_EQUALS_IGNORE_CASE), this::getRedundantCaseEqualsProblem)